README.md

```diff
@@ -27,7 +27,7 @@ the environment of the integration tests.
 
 ## Installation
 
-Pennyworth is tested on [openSUSE 13.1](http://en.opensuse.org/Portal:13.1).
+Pennyworth is tested on [openSUSE 13.2](https://en.opensuse.org/Portal:13.2) and [SLES12].
```

Comment:
Not everybody is familiar with the abbreviation SLES. Can you add a link to the homepage?
`[SUSE Linux Enterprise Server 12](https://www.suse.com/products/server/)`